openrr-planner/src/collision/urdf.rs

```diff
@@ -59,11 +59,11 @@ where
         }
         urdf_rs::Geometry::Mesh {
             ref filename,
-            scale,
+            ref scale,
         } => {
-            let scale = scale.unwrap_or(DEFAULT_MESH_SCALE);
+            let scale = scale.clone().unwrap_or(DEFAULT_MESH_SCALE);
```

Comment:
As well as https://github.com/openrr/urdf-viz/pull/207#discussion_r1178961498, clones in this PR will become unneeded once https://github.com/openrr/urdf-rs/pull/71 is released.